deploy/entrypoint.sh

```diff
@@ -4,11 +4,12 @@ set -e
 
 # on podman, host.containers.internal resolves to the host. this is equivalent to
 # host.docker.internal for docker.
-export QUIPUCORDS_API_URL="${QUIPUCORDS_API_URL:-https://host.containers.internal:9443}"
+export QUIPUCORDS_SERVER_URL="${QUIPUCORDS_SERVER_URL:-http://host.containers.internal:8000}"
```

Comment:
given discovery-server won't be exposed, we might need to change this one to
```suggestion
export QUIPUCORDS_SERVER_URL="${QUIPUCORDS_SERVER_URL:-http://discovery-server:8443}"
```